js/ui/yaxis-popup.js

```diff
@@ -107,5 +88,64 @@ Vue.component('yaxis-popup', {
         window.MetricQWebView.instances[0].setPlotRanges(false, true)
       }
     }
-  }
-})
+  },
+  mounted () {
+    const popupEle = document.querySelector('.yaxis_popup_div')
+    if (popupEle) {
+      const disablePopupFunc = () => {
+        this.$emit('toggle', false)
+        window.MetricQWebView.instances[0].reload()
+      }
+      veil.create(disablePopupFunc)
+      veil.attachPopup(popupEle)
+    }
+  },
+  methods: {
+    closePopup (evt) {
+      veil.destroy(evt)
+    },
+    closePopupModal: function (evt) {
+      if (evt.target.getAttribute('role') === 'dialog') {
+        veil.destroy(evt)
+      }
+    }
+  },
+  /* use vue-js for radio buttons */
+  template: `<div class="modal popup_div yaxis_popup_div" tabindex="-1" role="dialog" v-on:click="closePopupModal">
+    <div class="modal-dialog" role="document">
+    <div class="modal-content">
+    <popup-header v-bind:popupTitle="popupTitle"></popup-header>
+    <div class="modal-body">
+    <div class="form-check">
+    <input class="form-check-input" type="radio" value="global" name="yaxis" id="yaxis_global" v-model="yaxisRange" />
+    <label for="yaxis_global" class="form-check-label form-control-plaintext">Globales Min/Max</label>
+    </div>
+    <div class="form-check">
+    <input class="form-check-input" type="radio" value="local" name="yaxis" id="yaxis_local" v-model="yaxisRange" />
+    <label for="yaxis_local" class="form-check-label form-control-plaintext">Lokales Min/Max</label>
+    </div>
+    <div class="form-check">
+    <input class="form-check-input" type="radio" value="manual" name="yaxis" id="yaxis_manual" v-model="yaxisRange" />
+    <label for="yaxis_manual" class="form-check-label form-control-plaintext">Manuelles Min/Max</label>
+    </div>
+    <div class="form-group yaxis_popup_minmax">
+    <div class="form-group row">
+    <label for="yaxis_min" class="yaxis_popup_label_minmax col-sm-2 col-form-label">Min:</label>
+    <div class="col-sm-10"><input class="form-control" type="number" v-model="allMin" id="yaxis_min" :disabled.sync="manualDisabled" step="0.001"/></div>
+    </div>
+    <div class="form-group row">
+    <label for="yaxis_ax" class="yaxis_popup_label_minmax col-sm-2 col-form-label">Max:</label>
+    <div class="col-sm-10"><input class="form-control" type="number" v-model="allMax" id="yaxis_max" :disabled.sync="manualDisabled" step="0.001"/></div>
+    </div>
+    </div>
+    </div>
+    <div class="modal-footer">
+    <button class="btn btn-primary popup_ok" v-on:click="closePopup">
+    OK
+    </button>
+    </div>
+    </div>
+    </div>
+    </div>
+    </div>`
+}
```

Comment:
Can we have indentation in templates (automatically)?

Reply:
Automatically? No (see for example https://github.com/vuejs/eslint-plugin-vue/issues/1271). For this PR? sure